package.json

```diff
@@ -30,7 +30,7 @@
     "@backstage/core": "^0.6.0",
     "@backstage/plugin-catalog": "^0.2.9",
     "@backstage/theme": "^0.2.1",
-    "@material-ui/core": "^4.11.3",
+    "@material-ui/core": "^4.9.1",
```

Comment:
4.11.0 is used in core and the other plugins. Might be best to use that for the sake of consistency

Reply:
Will do

Reply:
done